src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -127,7 +127,9 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         bool IsReadOnly { get; }
 
         /// <summary>
-        /// True if the type is a record.
+        /// For source symbols, true if the type is a record.
+        /// For metadata symbols, true if the type is a record and a reference type.
+        /// Note: returns false for record structs in metadata since they don't have any distinctive marker.
```

Comment:
could be in a remarks section.

Reply:
Yes, this should be in a `remarks` section.